Podcast/EpisodeDetailViewController.swift

```diff
@@ -8,100 +8,95 @@
 
 import UIKit
 
-class EpisodeDetailViewController: ViewController, EpisodeDetailHeaderViewCellDelegate, UITableViewDelegate, UITableViewDataSource {
-    
-    var tableView: UITableView!
+class EpisodeDetailViewController: ViewController, EpisodeDetailHeaderViewDelegate {
+
+    let marginSpacing: CGFloat = EpisodeDetailHeaderView.marginSpacing
     var episode: Episode?
-    var comments: [Comment] = Array.init(repeating: Comment(episodeId: "", creator: "Mark Bryan", text: "Great point here!!", creationDate: "5 months ago", time: "5:13"), count: 10)
-    var episodeDetailViewHeaderHeight: CGFloat = 0
+    var headerView: EpisodeDetailHeaderView = EpisodeDetailHeaderView()
+    var episodeDescriptionView: UITextView = UITextView()
```

Comment:
I'm not sure a text view is what we want. If a description is too long the whole view (including the header) should scroll. So either the header and text view need to be put into a scroll view, or keep the table view logic.

Reply:
I disagree. I feel like the header should not scroll and just the description should. Playing with the UI I think this looks much better. We can talk to eileen about it on Sunday and I will retroactively make changes if we want to change this.